content/sensu-go/6.2/operations/control-access/ldap-auth.md

```diff
@@ -144,6 +144,56 @@ spec:
 
 {{< /language-toggle >}}
 
+**Example LDAP configuration: Use `memberOf` attribute instead of `group_search`**
+
+LDAP automatically returns a `memberOf` attribute in users' accounts.
```

Comment:
```suggestion
If your LDAP server is configured to return a `memberOf` attribute when you perform a query, you can use `memberOf` in your Sensu LDAP implementation instead of `group_search`.
```

Reply:
^ @palourde is this accurate? I'm sorry, I'm not sure if you mean users have to do something in LDAP to use memberOf in Sensu.

Reply:
Yep, that's perfect!